IlmBase/CMakeLists.txt

```diff
@@ -158,35 +163,26 @@ IF (NAMESPACE_VERSIONING)
   SET ( ILMBASE_LIBSUFFIX "-${ILMBASE_VERSION_API}" )
 ENDIF ()
 
-SET_TARGET_PROPERTIES ( Half
-  PROPERTIES
-  VERSION 12.0.0
-  SOVERSION 12
-  )
-SET_TARGET_PROPERTIES ( Iex
-  PROPERTIES
-  VERSION 12.0.0
-  SOVERSION 12
-  OUTPUT_NAME "Iex${ILMBASE_LIBSUFFIX}"
-  )
-SET_TARGET_PROPERTIES ( Imath
-  PROPERTIES
-  VERSION 12.0.0
-  SOVERSION 12
-  OUTPUT_NAME "Imath${ILMBASE_LIBSUFFIX}"
-  )
-SET_TARGET_PROPERTIES ( IlmThread
-  PROPERTIES
-  VERSION 12.0.0
-  SOVERSION 12
-  OUTPUT_NAME "IlmThread${ILMBASE_LIBSUFFIX}"
-  )
-SET_TARGET_PROPERTIES ( IexMath
-  PROPERTIES
-  VERSION 12.0.0
-  SOVERSION 12
-  OUTPUT_NAME "IexMath${ILMBASE_LIBSUFFIX}"
-  )
+SET (ILMBASE_LIBRARY_NAMES Half Iex Imath IlmThread IexMath)
+FOREACH (LIBNAME ${ILMBASE_LIBRARY_NAMES})
+  IF (LIBNAME STREQUAL "Half")
```

Comment:
Remind me why we’re not suffixing half like all the other libs if the suffix is enabled? Seems like it should be consistent